gnosis/safe-react
# Removed pinned Safe Apps from localStorage when they are not included in the Remote Apps List

## What it solves
Resolves #2847

## How this PR fixes it
Removed pinned Safe Apps from localStorage when they are not included in the Remote Apps List

## How to test it
1. Add a new visible app in Django
2. Bookmark it as favorite in the apps section
3. Remove the app from Django
4. Go back to the apps section and refresh the page
5.  inspect local storage and the Safe App id should be removed
